packages/core/src/components/toast/toaster.tsx

```diff
@@ -101,6 +109,7 @@ export class Toaster extends AbstractPureComponent2<IToasterProps, IToasterState
     public static defaultProps: IToasterProps = {
         autoFocus: false,
         canEscapeKeyClear: true,
+        maxToasts: 5,
```

Comment:
this is technically a breaking change in behavior. I think you should not set a default, and if `maxToasts === undefined`, then do not enforce a limit.